src/models/events/AutocompleteEvent.ts

```diff
@@ -0,0 +1,9 @@
+import { AnalyticsEventType } from '../AnalyticsEventType';
+import { EnumOrLiteral } from '../utils';
+
+/** Event for autocomplete selection. */
+export interface AutocompleteEvent {
```

Comment:
I think in the SDK we send `queryId` with this (even if it's not required)?

Reply:
sure thing, updated to include it optionally. I believe visitor field is also included in SDK analytics events from the few events I just tested. Made a separate item for that